server/bin/pbench-backup-tarballs.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+import os
+import sys
+import sh
+import glob
+import signal
+import configtools
+import subprocess
+from argparse import ArgumentParser
+from configparser import ConfigParser, Error as ConfigParserError, NoSectionError, NoOptionError
```

Comment:
I don't think we need configparser at all any longer: it's all hidden inside PbenchConfig.